lib/ex_json_schema/validator/format.ex

```diff
@@ -61,12 +61,23 @@ defmodule ExJsonSchema.Validator.Format do
     end
   end
 
-  defp do_validate(_, "date-time" = format, data) do
+  defp do_validate(root, "date-time" = format, data) do
     data
     |> String.upcase()
     |> DateTime.from_iso8601()
```

Comment:
Wouldn't it be possible to get the same behavior by only changing this line? `NaiveDateTime.from_iso8601/1` simply ignores the timezone as far as I can tell 🤔 
```suggestion
    |> NaiveDateTime.from_iso8601()
```

Reply:
Yes, I realized only later. Submitting a separate PR